frame/utility/src/lib.rs

```diff
@@ -266,7 +268,13 @@ pub mod pallet {
 				let result = if is_root {
 					call.dispatch_bypass_filter(origin.clone())
 				} else {
-					call.dispatch(origin.clone())
+					let mut filtered_origin = origin.clone();
```

Comment:
There is no need to clone, right? You can just move the value as it is not used later.

Reply:
There is a loop